docs/internals/contributing/writing-documentation.txt

```diff
@@ -117,6 +96,103 @@ The documentation is organized into several categories:
   hesitate to refer the reader back to the appropriate tutorial rather than
   repeat the same material.
 
+How to start contributing documentation
+=======================================
+
+Clone the Django repository to your local machine
+-------------------------------------------------
+
+If you'd like to start contributing to our docs, get the development version of
+Django from the source code repository (see
+:ref:`installing-development-version`):
+
+.. console::
+
+     $ git clone https://github.com/django/django.git
+
+If you're planning to submit these changes, you might find it useful to make a
+fork of the Django repository and clone this fork instead.
+
+Set up a virtual environment and install dependencies
+-----------------------------------------------------
+
+Create and activate a virtual environment, then install the dependencies:
+
+.. console::
+
+     $ python -m venv .venv
+     $ source .venv/bin/activate
```

Comment:
`...\> source .venv\bin\activate` will not work on Windows, so it's probably better to use `bash` instead of `console`.

Reply:
I see the Docs / docs CI failed. I'm still unfamiliar with this part of the reST syntax. Does this need to be a `code-block` directive? For example:

```rst
.. code-block:: bash
```

Reply:
Sorry my mistake, it should be `shell`.